client/my-sites/patterns/components/get-started/style.scss

```diff
@@ -0,0 +1,69 @@
+@import "@automattic/components/src/styles/typography";
+@import "@wordpress/base-styles/breakpoints.scss";
+
+.patterns-get-started__body {
+	.patterns-get-started__buttons {
+		padding: 0 110px;
+		margin-top: -32px;
+		margin-bottom: 48px;
+
+		.patterns-get-started__buttons-build {
+			font-family: $font-sf-pro-text;
+			font-size: rem(13px);
+			color: var(--studio-white);
+			height: 40px;
+			padding: 0 17px;
+			border-radius: 2px;
+			background: #3858e9;
+		}
+	}
+
+	.patterns-get-started__list {
+		overflow: auto;
+
+		.patterns-get-started__list-inner {
+			min-width: 1220px;
+			padding: 0 110px;
+			display: flex;
+			gap: 24px;
+
+			.patterns-get-started__item {
+				flex: 1;
+
+				.patterns-get-started__item-image {
+					width: 100%;
+					padding-bottom: 16px;
+				}
+
+				.patterns-get-started__item-name {
+					font-family: $font-sf-pro-text;
+					font-size: rem(12px);
+					color: #a7aaad;
+					font-weight: 500;
+					line-height: 1.6;
+					letter-spacing: 0.4px;
+					padding-bottom: 16px;
+				}
+
+				.patterns-get-started__item-description {
+					font-family: $font-recoleta;
+					color: var(--studio-white);
+					font-size: rem(24px);
+					line-height: 1.3;
+					padding-bottom: 16px;
```

Comment:
```suggestion
```
Looking at the Figma design, I don't think we should actually have any `padding-bottom` for this element.